playbook/app/pb_kits/playbook/pb_currency/_currency.scss

```diff
@@ -24,6 +24,29 @@
   [class^=pb_currency_wrapper] {
     display: flex;
 
+    &[class*=_deemphasized] [class^=pb_body_kit][class*=_light] {
+      color: $text_lt_default;
+    }
+
+    &[class*=_deemphasized] [class^=pb_title_kit][class*=_4] {
+      color: $text_lt_default;
+      font-weight: 400;
+    }
+
+    &[class*=_light] [class^=pb_body_kit][class*=_light] {
+      color: $text_lt_light;
+    }
+
+    &[class*=_light] [class^=pb_title_kit][class*=_4] {
+      color: $text_lt_light;
+      font-weight: 400;
+    }
+
+    &[class*=_bold] [class^=pb_body_kit][class*=_light] {
+      color: $text_lt_default;
+      font-weight: 700;
```

Comment:
Should be able to use `$bolder` sass variable defined in `_typography.scss` here for the `font-weight`.

Reply:
Thanks for the suggestion. I implemented it and it's working great. Same for `$regular`.